src/Components/RecsList/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { lazy, Suspense } from 'react';
+import { useIntl } from 'react-intl';
+
+import Main from '@redhat-cloud-services/frontend-components/Main';
+import PageHeader, {
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+
+import Loading from '../Loading/Loading';
+import messages from '../../Messages';
+
+const RecsListTable = lazy(() =>
+  import(/* webpackChunkName: 'RulesTable' */ '../RecsListTable/')
+);
+
+const RecsList = () => {
+  const intl = useIntl();
+  return (
```

Comment:
Do we want to implement error boundaries https://reactjs.org/docs/code-splitting.html#error-boundaries in the Advisor?